platform/broadcom/sonic-platform-modules-inventec/d7332/modules/sff_spec.h

```diff
@@ -0,0 +1,415 @@
+#ifndef __SFF_SPEC_H
```

Comment:
use optoe driver

Reply:
inv_sff is used to get "lpmode", "presence" info that optoe does not support
optoe2 is used to get "eeprom" info

Please refer to "inventec_d7332_util.py" & "sfputil.py" code snippets as shown below:
File: platform/broadcom/sonic-platform-modules-inventec/d7332/utils/inventec_d7332_util.py
=========
Code snippet
=========
Line165: status, output =exec_cmd("echo optoe2 0x50 > /sys/bus/i2c/devices/i2c-1/i2c-6/i2c-"+str(i)+"/new_device", 1)

File: device/inventec/x86_64-inventec_d7332-r0/plugins/sfputil.py
=========
Code snippet
=========
Line103: eeprom_path = "/sys/bus/i2c/devices/{0}-0050/eeprom"

Reply:
@lguohan please review the comment above. Thanks!